Hourglass/AppEntry.cs

```diff
@@ -91,6 +107,16 @@ protected override bool OnStartup(StartupEventArgs eventArgs)
     protected override void OnStartupNextInstance(StartupNextInstanceEventArgs eventArgs)
     {
         CommandLineArguments arguments = CommandLineArguments.Parse(eventArgs.CommandLine);
+        if (arguments.ValidateArgs && !arguments.HasParseError)
+        {
+            Console.WriteLine("true");
+            return;
+        } else if (arguments.ValidateArgs && arguments.HasParseError)
+        {
+            Console.WriteLine("false");
+            return;
+        }
+        

```

Comment:
Can be refactored by adding bool param which disables timers printing passed to `ArgsValidationFailedValidateArgs` (see comment above).